requirements.txt

```diff
@@ -10,4 +10,6 @@ grpcio>=1.33.2
 grpcio-tools>=1.33.2
 protobuf>=3.12.2
 configargparse>=1.2.3
+multiaddr==0.0.9
+pymultihash==0.8.2
```

Comment:
We usually have restricions of the form >= to support all dependency versions starting from minimally possible, otherwise we'll have lots of unnecessary conflicts with other libraries